test/RoadRegistry.Tests/Framework/Projections/EditorContextScenarioExtensions.cs

```diff
@@ -265,6 +265,8 @@ public static async Task ExpectNone(this ConnectedProjectionScenario<EditorConte
             }
         }
 
+        //IMPORTANT: Each time you change the db sets on the context, you must adjust this method as well.
```

Comment:
There might be a way using reflection to do this, I just don't have the time to do so ATM.